scripts/entity/tactical/humans/rf_billman_heavy.nut

```diff
@@ -27,7 +27,7 @@ this.rf_billman_heavy <- ::inherit("scripts/entity/tactical/human" {
 
 		this.m.Skills.add(::new("scripts/skills/perks/perk_battle_forged"));
 		this.m.Skills.add(::new("scripts/skills/perks/perk_rotation"));
-		this.m.Skills.add(::new("scripts/skills/perks/perk_rf_follow_up"));
+		this.m.Skills.add(::new("scripts/skills/perks/perk_rf_follow_up")); // Disabled due to crashing the game on a follow up kill
```

Comment:
you didn'T actually disable/remove/replace the skill here

Reply:
whoops